src/test/java/jewellery/inventory/helper/UserTestHelper.java

```diff
@@ -0,0 +1,76 @@
+package jewellery.inventory.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+import java.util.UUID;
+import jewellery.inventory.dto.UserRequest;
+import jewellery.inventory.dto.UserResponse;
+import jewellery.inventory.model.User;
+
+public class UserTestHelper {
+  public static User createTestUser() {
+    User user = new User();
+    user.setName("John");
+    user.setEmail("john@example.com");
```

Comment:
I noticed some of the values are duplicated within the file. Let's extract them in constants if that's the case.

Reply:
Fixed